README.rst

```diff
@@ -98,7 +98,7 @@ API Client
 
 .. code-block:: python
 
-    from pybigquery.api import ApiClient
+    from sqlalchemy_bigquery.api import ApiClient
     api_client = ApiClient()
     print(api_client.dry_run_query(query=sqlstr).total_bytes_processed)
```

Comment:
We can remove this code sample now that we don't have an `ApiClient`

Reply:
Dang, I did yesterday, but I guess that change lost in my docfx flailing.  Thanks, will do.

Reply:
done